### PR TITLE
Javalab: fix editor resize ellipsis location and add asset manager button to Theater

### PR DESCRIPTION
**Pull request by bencodeorg**
1. Fix editor resizer ellipsis to appear in correct place.
2. Add asset manager button to visualization area in Theater. Hide "Preview" button here (per input from Moneppo).
3. Renames `TheaterVisualization` to `TheaterVisualizationColumn` to be consistent with Neighborhood.

**Before (Theater)**

image

**After (Theater)**

image

**Before (Neighborhood)**

image

**After (Neighborhood)**

image

## Testing story

Tested manually locally.

## Follow-up work

1. There are some issues with centering of headers (made a little worse in this PR for the Preview panel by removing a placeholder span for a future "full size" feature)  -- should "Console", "Preview", and "Editor" headers be centered across the entire width of the header, or in the space between any buttons that appear on the left and right sides of the header? It looks like we use the latter currently, but I think the former maybe makes more sense?
2. There's a few pixel "jump" when you use the editor resizer now that I couldn't quite figure out. We have a [Jira task to make the Console also resize when resizing the Editor](https://codedotorg.atlassian.net/browse/CSA-446), so I thought we could resolve that issue there.

### CODE REVIEW

**Review by molly-moen — APPROVED** [review]
LGTM! I think we should go for centering the titles within the entire header instead of between buttons.

File: apps/src/javalab/PreviewPaneHeader.jsx [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
   PaneButton
 } from '@cdo/apps/templates/PaneHeader';
 import CollapserIcon from '@cdo/apps/templates/CollapserIcon';
 
-export default function PreviewPaneHeader({isCollapsed, isFullscreen}) {
+export default function PreviewPaneHeader({
+  isCollapsed,
+  isFullscreen,
+  showAssetManagerButton = false,
+  showPreviewTitle = true
```
Comment (maddiedierker) [inline apps/src/javalab/PreviewPaneHeader.jsx:14]: nit: set these as default props below the propTypes definition, e.g.:

```javascript
PreviewPaneHeader.propTypes = {};

PreviewPaneHeader.defaultProps = {
  showAssetManagerButton: false,
  showPreviewTitle: true
};
```

Comment (bencodeorg) [reply apps/src/javalab/PreviewPaneHeader.jsx:14]: 👍 is that our preferred style? I wasn't sure and was googling and found this when I was deciding which to do:

https://stackoverflow.com/questions/47774695/react-functional-component-default-props-vs-default-parameters

Comment (maddiedierker) [reply apps/src/javalab/PreviewPaneHeader.jsx:14]: ooo, interesting -- TIL! i mostly see us using `defaultProps` for functional components, but if that will be deprecated, let's keep this as-is and we'll start transitioning to default params

**Review by maddiedierker — APPROVED** [review]

**Pull request merged**